src/index.js

```diff
@@ -3,7 +3,21 @@ const core = require('@actions/core')
 
 const { runAction } = require('./release-notify-action')
 
+const actionRef = process.env.GITHUB_ACTION_REF
+
 async function run() {
+  if (actionRef === 'main' || actionRef === 'master') {
+    core.warning(
+      `nearform/github-action-notify-release is pinned at HEAD. We strongly ` +
+        `advise against pinning to "@master" as it may be unstable. Please ` +
+        `update your GitHub Action YAML from:\n\n` +
+        `    uses: 'nearform/github-action-notify-release@master'\n\n` +
+        `to:\n\n` +
+        `    uses: 'nearform/github-action-notify-release@v0'\n\n` +
```

Comment:
v0 is wrong here. The latest major is v1. I don't like though that we're using a specific version because when we release a new major we have to remember to change this, but I'm not sure what better options we have

Reply:
I have added a function which gets the latest major tag and uses that in the template.

Reply:
I tested this approach on another repo and it doesn't work. I will look for a better solution

Reply:
yeah I was looking at the code and I wasn't very convinced either. there's a compromise to accept here, using an automated way to do this is not necessarily the way to do it

Reply:
Since this is just a warning do you think we can just use a placeholder like

`uses: 'nearform/github-action-notify-release@<latest-major-version>`

Reply:
this is not accurate either. users can reference any valid release or tag

Reply:
ok so maybe use something like
`uses: 'nearform/github-action-notify-release@<release/tag version>`